suite/tests/CMakeLists.txt

```diff
@@ -5567,6 +5567,38 @@ if (BUILD_CLIENTS)
     # b/c our tests have too-new instrs for older machines.
   endif (X86)
 
+  ###########################################################################
+  # drpoints tests
+
+  if (X86 OR AARCH64)
+    if (NOT WINDOWS)
+      torunonly_ci(tool.drpoints.simple ${ci_shared_app} drpoints
+        client-interface/${ci_shared_app} # for .expect
```

Comment:
So this doesn't test that drpoints did anything. Please add an output template for a sanity check that drpoints actually wrote something. It may be nicer to not print to stdout and instead write the file and add a CMake script checking the file actually exists: so there's an end to end test on the output file being written.